spec/models/mechanic_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Mechanic, type: :model do
+  describe "relationships" do
+    it { should belong_to(:amusement_park) }
+    it { should have_many(:mechanics_rides) }
+    it { should have_many(:rides) }
+  end
```

Comment:
It looks like you might have a `belongs_to` test that you do not need here.

You're also missing the `through` for line 7.